Tribler/Core/Utilities/utilities.py

```diff
@@ -167,7 +167,7 @@ def isValidTorrentFile(metainfo):
     try:
         validTorrentFile(metainfo)
         return True
-    except:
+    except ValueError:
         print_exc()
```

Comment:
This shouldn't print stuff (I know it's not related to this PR, but maybe you want to create a new one fixing this to use the logging system).